spring-integration-core/src/main/java/org/springframework/integration/config/xml/PointToPointChannelParser.java

```diff
@@ -30,6 +28,7 @@
 import org.springframework.integration.store.MessageGroupQueue;
 import org.springframework.util.StringUtils;
 import org.springframework.util.xml.DomUtils;
+import org.w3c.dom.Element;
```

Comment:
Hello, @manueljordan !

Thank you for update!

As you already noticed Gary's comment about HTTP tests, you should `rebase` to the latest master and "un"`@Ignore` those tests in the `DefaultHttpHeaderMapperFromMessageOutboundTests`.

But that's not all. I'll let you know what to change else during this review phase.

So, this one imports order change is wrong. See https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style#import-statements. 
Right, it is minor and I can fix on merge. But to follow with our style helps minimize the code for review and save some time for all of us. At least I'll stop to write similar comments here :smile: